pyccl/tracers.py

```diff
@@ -180,6 +180,31 @@ def __init__(self):
         # Do nothing, just initialize list of tracers
         self._trc = []
 
+    def __bool__(self):
+        return bool(self._trc)
+
+    @property
+    def chi_min(self):
+        """Return ``chi_min`` for this ``Tracer``. If it contains more than
+        one tracers and their ``chi_mins`` are different it will raise an
+        ``AttributeError``.
+        """
+        chis = [tr.chi_min for tr in self._trc]
+        if chis.count(chis[0]) != len(chis):
+            raise AttributeError("Tracers have different chi_min.")
+        return chis[0]
```

Comment:
Why not just remove `chi_arr` from an attribute of `Tracer`? I don't think it's ever used. Returning the minimum of the minima might result to ambiguous behavior, I think better to be on the safe side and just throw an error if that ever happens.

Reply:
wait, I phrased my suggestion above very badly. I think this function/property would be super useful if it returned the minimum of all minima. The idea is that what you usually do with these tracers is use their kernels to integrate over when calculating power spectra, trispectra etc. So `chi_min` and `chi_max` tell you the range over which you need to integrate (and they also tell you about the scales that are relevant, since k=(ell+1/2)/chi). Since a given tracer may be a linear combination of several kernels, what you want is the minimum chi of all those.

So could we do that?

Reply:
That's a smart use actually, I implemented it.